recipes/wibl-python/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "wibl-python" %}
+{% set version = "1.0.3" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CCOMJHC/WIBL/archive/refs/tags/{{ name }}-{{ version }}.tar.gz
+  sha256: d4e6d4b50f0f674657c817531e34c8d29056ea1d6ec121bf54f4f8e32ea42012
+
+build:
+  entry_points:
+    - wibl = wibl.command.__main__:main
+  noarch: python
+  script: |
+    {{ PYTHON }} -m pip install ./wibl-python/wibl-manager
+    {{ PYTHON }} -m pip install csbschema==1.1.2
```

Comment:
You will have to create another recipe for `csbschema` as we do not permit installing for `pypi` directly.

Reply:
Hi @synapticarbors. Thanks for catching this. Yeah, I came to that conclusion. Grayskull to the rescue...

Reply:
@synapticarbors Added csbschema conda-forge dependency, waiting on CI. Thanks for reviewing these quickly!